packages/@aws-cdk/aws-servicecatalog/lib/tagoptions.ts

```diff
@@ -0,0 +1,35 @@
+import * as cdk from '@aws-cdk/core';
+import { CfnTagOption } from '.';
+import { hashValues } from './private/util';
+import { InputValidator } from './private/validation';
+
+/**
+ * Defines a Tag Option, which are similar to tags
+ * but have multiple values per key.
+ */
+export class TagOptions {
+  /**
+  * List of CfnTagOption
+  */
+  public readonly cfnTagOptions: CfnTagOption[];
+
+  constructor(stack : cdk.Stack, tagOptionsMap: {[key: string]: string[]}) {
```

Comment:
This is not really an API we would consider adding to the CDK - it's very unnatural.

Get rid of the `stack` parameter from here.